src/Dfc.CourseDirectory.Services/Models/Venues/Venue.cs

```diff
@@ -81,22 +81,22 @@ public class Venue
 
             if (string.IsNullOrWhiteSpace(venueName))
             {
-                throw new ArgumentException("message", nameof(venueName));
+                throw new ArgumentException("Error creating Venue: Required field is null or whitespace", nameof(venueName));
```

Comment:
How about `"Null or whitespace {nameof(venueName)}.`"?

Reply:
Or just something that is descriptive of the error.

Reply:
The `nameof` is already rolled up into the string by `ArgumentException`, is there a reason for repeating the value in the `message` param?

![Selection_20210129-01-831538315383153](https://user-images.githubusercontent.com/19378/106258641-02295d80-6216-11eb-829c-86b5480e99d6.png)